drivers/modem/Kconfig.gsm

```
@@ -36,6 +36,24 @@ config MODEM_GSM_QUECTEL
 
 endchoice
 
+choice MODEM_GSM_STATUS_COMMAND_TYPE
```

Then this can just be 
```suggestion
choice MODEM_GSM_STATUS_COMMAND
```